content/docs/ops/runbook/troubleshooting-kubernetes.md

```diff
@@ -14,25 +14,36 @@ Alerts are generated whenever a pod's status is not `Running`. Alerts contains t
 
 If the namespace is `default` and the pod name is a random string then it is a managed service and likely only impacts a specific tenant application.  If the namespace is `kube-system` or the pod name is human readable then it is a platform component which could impact all managed services.
 
-#### Fixing non-running pods
-Login to a kubernetes master and describe the pod:
+#### Login to kubernetes master
+
+From a jumpbox in the appropriate environment:
+- ssh to a master
+- update the PATH to find the kubernetes binaries
+- sudo to root for all the `kubectl` commands
+
 ```sh
-/var/vcap/packages/kubernetes/bin/kubectl --namespace :namespace describe pod :pod-name
+bosh -d kubernetes ssh master/0
+PATH=$PATH:/var/vcap/packages/kubernetes/bin
+sudo bash
 ```
 
-The `Events` section should indicate why the pod cannot be started.  Resolve the underlying issue and the pod should transition into a `Running` state.  If you need to force the pod to be restarted you can delete it which will cause kubernetes to immediately recreate it.
-```sh
-/var/vcap/packages/kubernetes/bin/kubectl --namespace :namespace delete pod :pod-name
+#### Fixing non-running pods
+Login to a kubernetes master, find the namespaces, and describe the pods:
+```bash
+kubeclt get namespaces 
+kubectl --namespace :namespace describe pod :pod-name
 ```
+where you provide values for :namespace and :pod-name, where a pod-name is
+will be something like `xc956b1d94dd64-master-0`
 
+The `Events` section should indicate why the pod cannot be started.  Resolve the underlying issue and the pod should transition into a `Running` state.  If you need to force the pod to be restarted you can delete it which will cause kubernetes to immediately recreate it:
```

Comment:
This is potentially misleading. Kubernetes will recreated deleted pods that are part of a statefulset / deployment / daemonset / etc, but it's also possible to run pods that aren't managed by one of these resources. May want to add that caveat or drop this note.